server/src/main/java/io/crate/metadata/doc/DocTableInfo.java

```diff
@@ -816,7 +817,7 @@ private void updateParentsInnerTypes(ColumnIdent column,
 
     public DocTableInfo dropConstraint(String constraint) {
         List<CheckConstraint<Symbol>> newConstraints = checkConstraints.stream()
-            .filter(x -> !x.name().equals(constraint))
+            .filter(x -> !constraint.equals(x.name()))
             .toList();
```

Comment:
just curious, what's the difference here?

Reply:
x.name() is marked as `Nullable`.